### ISSUE TITLE
Support for `include_str`?

### ISSUE THREAD

**Issue by Testare** [mersinvald/aquamarine#19]
I think this is a great project. I was trying to use aquamarine so I could have diagrams for some high-level designs in my crate, and given the potential complexity of this I wanted to write this documentation in its own file and add it to the module like so
```
#[cfg_attr(doc, aquamarine::aquamarine)]
#[doc = include_str!("../foobar.md")]
```

I've also tried
```
#[cfg_attr(doc, aquamarine::aquamarine, doc = include_str!("../foobar.md"))]
```

I've also tried rearranging the order but it seems not to matter.

However, in both cases what happens is that the file is included, and the mermaid code blocks are rendered as regular code blocks instead of diagrams. I'm a relatively new rustacean with no macro writing experience, is there a way to accomplish this?

**Comment by mersinvald** [comment]
Hi @Testare, this case has to be supported by a macro, there's no way to do it without modifying aquamarine

**Comment by drbh** [comment]
@Testare this is not exactly the feature you specified above, but this PR #26 is how I'm including the contents of an external file (it can probably be modified for the API you specified if this does not meet your usecase). 

```rust
#[cfg_attr(doc, aquamarine, path = "./diagram.md")]
```

Hope this is helpful!

**Comment by mersinvald** [comment]
Fixed in #26, huge thanks to @drbh

**Comment by matrach** [comment]
Could you please reopen this issue? #26 and current `include_mmd!` is about including a *single diagram*, rather than the *whole documentation* block refactored out to a separate file as indicated by the OP. 

Would it be possible for aquamarine to support such a workflow?

The files would have a structure like this:

```
- crate/
- - src/
- - - lib.rs
- - - module/
- - - - README.md
- - - - ...
```

in README.md:
````
# Section with a diagram
```mermaid
 graph LR
     s([Source]) --> a[[aquamarine]]
     r[[rustdoc]] --> f([Docs w/ Mermaid!])
     subgraph rustc[Rust Compiler]
     a -. inject mermaid.js .-> r
     end
```
# Some other details
````

in lib.rs

```rust
#[aquamarine]
/// Header line for rustdoc
///
#[doc = include_str!("module/README.md")]
mod module {
   ...
}
```
